concordium-sdk/src/main/java/com/concordium/sdk/responses/ProtocolVersion.java

```diff
@@ -0,0 +1,42 @@
+package com.concordium.sdk.responses;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.val;
+
+import java.util.*;
+
+public enum ProtocolVersion {
+    V1,
+    /**
+     * https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P2.txt
+     */
+    V2,
+    /**
+     * https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P3.txt
+     */
+    V3,
+    /**
+     * https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P4.txt
+     */
+    V4;
+
+    private static final List<ProtocolVersion> protocolVersions = new ArrayList<>();
+
+    static {
+        protocolVersions.add(null);
```

Comment:
What is this for?

Reply:
´protocolVersions´ is a zero indexed list and we start with P1.
Related to the json parsing of the protocol version: 
```
@JsonCreator
    public static ProtocolVersion forValue(int protocolVersion) {...}
```
Only [1-4]  is returned.

Reply:
Made it a bit more explicit now and attached a comment.